packages/core-app-api/api-report.md

```diff
@@ -227,6 +227,9 @@ export type AppRouteBinder = <
   >,
 ) => void;
 
+// @public
+export function AppRouter({ children }: { children?: ReactNode }): JSX.Element;
```

Comment:
Plz unpack props inside the component :)

Reply:
Hah, I even declared `AppRouterProps` but didn't use it x)